### ISSUE TITLE
Add shortcut to fold/unfold subfloors without having to navigate to the preferences

### ISSUE THREAD

**Issue by chtitdraco** [ClaudeMetz/FactoryPlanner#299]
### Suggestion

Following the removal of the button toggling between folded / unfolded subfloors, could you add a shortcut to perform the same action?
This would save several clicks (currently need to navigate to the preferences and (un)checking the checkbox there.

**Comment by ClaudeMetz** [comment]
Yeah this is something I should probably do, appreciate the suggestion.

**Comment by chtitdraco** [comment]
Thanks!
